TodoApp/src/TodoApp.Api/Web.Release.config

```diff
@@ -7,13 +7,10 @@
     In the example below, the "SetAttributes" transform will change the value of 
     "connectionString" to use "ReleaseSQLServer" only when the "Match" locator 
     finds an attribute "name" that has a value of "MyDB".
-    
-    <connectionStrings>
-      <add name="MyDB" 
-        connectionString="Data Source=ReleaseSQLServer;Initial Catalog=MyReleaseDB;Integrated Security=True" 
-        xdt:Transform="SetAttributes" xdt:Locator="Match(name)"/>
-    </connectionStrings>
-  -->
+   -->
+  <connectionStrings xdt:Transform="Replace">
+    <add name="DefaultConnection" connectionString="" />
```

Comment:
I've actually liked the `mongo_list_connection` (a name lost in between commits) a bit more to be honest…